Cargo.toml

```diff
@@ -39,7 +39,6 @@ debug = 0
 default = []
 
 [workspace]
-resolver = "2"
```

Comment:
Resolver 2 is default with edition 2021 so this doesnt change anything.

https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html